contributors/design-proposals/propagation.md

```diff
@@ -150,6 +165,42 @@ non-privileged', an environment check and an WARNING log will be emitted about
 whether propagation mode is supported.
 
 
+## Extra Concerns
+
+@lucab and @euank has some extra concerns about pod isolation when propagation
+modes are changed, listed below:
+
+1. how to clean such pod resources (as mounts are now crossing pod boundaries,
```

Comment:
See the last paragraph of https://github.com/kubernetes/community/pull/193#issuecomment-267968938 in this same thread for clarification. This is talking about cleaning bind-mounts that can be propagated across different pods, and as such can keep other subtrees busy.